# Physical robot parameters
v_max: 1.0
a_max: 2.0
j_max: 8.0
w_max: 1.8
anga_max: 6.283 # angular acceleration

